packages/@aws-cdk/aws-ec2-alpha/lib/route.ts

```diff
@@ -175,6 +176,47 @@ export interface NatGatewayProps extends NatGatewayOptions {
   readonly vpc?: IVpcV2;
 }
 
+/**
+ * Properties to define a VPC peering connection.
+ */
+export interface VPCPeeringConnectionProps {
```

Comment:
Question on the naming here, is the convention for other VPC related interfaces to use `VPCSomethingSomething` in all caps or `VpcSomethingSomething`?

Reply:
I was actually just about to push changes but I realized that the L1 construct is [CfnVPCPeeringConnection](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ec2.CfnVPCPeeringConnection.html) so I believe using `VPCSomethingSomething` would make the naming more consistent.

Reply:
Sounds good, I'm okay with keeping it as is then if it's consistent with the L1.